src/platform/silabs/SiWx917/wifi_args.gni

```diff
@@ -25,7 +25,7 @@ arm_platform_config = "${efr32_sdk_build_root}/efr32_arm.gni"
 
 mbedtls_target = "${efr32_sdk_build_root}:efr32_sdk"
 
-chip_crypto = "platform"
+chip_crypto = "mbedtls"
```

Comment:
here as well, better to remove args.gni and rename this as args.gni
Plus the chip_crypto should be platform here